### DISCUSSION TITLE
ReferenceError: Node is not defined

### DISCUSSION THREAD

**Discussion by mohamilin** [SantriHub/app#11] [Q&A]
mas @zainfathoni @thoriqaziz 
Ketika saya install dan pakai Popover muncul error spt ini. Kira-kira ada saran / refrensi ?
saya pake node v16.15.0
terima kasih

![image](https://user-images.githubusercontent.com/50976240/177022645-4336de58-41a2-405e-9792-6ab2f6c794a1.png)
![image](https://user-images.githubusercontent.com/50976240/177022651-8fb4a4df-2526-410e-9830-da256f62ec18.png)

**Comment by zainfathoni** [comment]
Please push your branch to the repository and create a **Draft** PR.
I'll try to reproduce it on my side.

**Reply by mohamilin** [reply]
oke thanks mas @zainfathoni
